### PR TITLE
grid: various fixes and tweaks

### PR DESCRIPTION
**Pull request by arthyn**
Fixes all these issues in addition to adding a temporary favicon:

- dark mode
	- [X] sigils not dark mode friendly
	- [X] installing tiles have wrong text color or too light
- mobile
	- [X] missing system menu
	- [X] bottom padding wrong
- tiles 
	- [X] static size until mobile?
	- [X] grid should be centered when the first row isn't full
- nav
	- [X] can't click through items if dialog/menu open
	- [X] system menu open doesn't dim other items
- [X] system prefs not respecting height
- [X] system prefs toggle alignment
- [X] need to ignore sig for better autocomplete
- [X] keyboard selection of devs doesn't add to recent list
- [X] alt-primary button needs different focus-ring color

### CODE REVIEW

**Comment by arthyn** [comment]
here's a link to the deployed mock version: https://landscape-tweaks.vercel.app/

@urcades @g-a-v-i-n if y'all want to take a look

**Review by liam-fitzgerald — APPROVED** [review]
LGTM! Pending design approval

**Comment by fischerando** [comment]
Idk if this is the place for this kind of feedback, but the comma on the "Suspend" screen is ungrammatical.

**Comment by urcades** [comment]
> Idk if this is the place for this kind of feedback, but the comma on the "Suspend" screen is ungrammatical.

Always appreciate feedback like this! Thanks for providing @fischerando 

Now, for my own two feedback items:

### 1. Mobile "Touch Icons" Missing

This is a small one, and I'm only including it due to the mention of us situating a placeholder favicon with this latest work:

IMG_3633

On iOS, it looks like we'll need to provide a custom "apple touch" image in order for a nice "app tile"-like representation of Grid to appear on the home screen. Apple describes how to specify this out [here](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html), but there's also a useful [stackoverflow article](https://stackoverflow.com/questions/1951381/configuring-android-web-applications) which spells out four possible apple-specific configurations we could make to our web app to make it feel more 'native':

```
<link rel="apple-touch-icon" href="/custom_icon.png"/>                        // necessary
<link rel="apple-touch-startup-image" href="/startup.png">                    // optional, but cool
<meta name="apple-mobile-web-app-capable" content="yes" />                    // necessary(?)
<meta name="apple-mobile-web-app-status-bar-style" content="black" />         // necessary
```

Android has three related fields we should probably afford for, for max coverage:

```
<meta name="mobile-web-app-capable" content="yes">
<link rel="shortcut icon" sizes="196x196" href="youriconhighres.png">
<link rel="shortcut icon" sizes="128x128" href="youricon.png">
```

### 2. Mobile "Proto-Leap" field focus bugginess

This feedback is best explained via a video. Please watch this to see what I'm encountering:

https://user-images.githubusercontent.com/1195363/133638415-4b0c760a-6127-4853-88ae-26e7616420eb.MP4

In a nutshell, it looks like when we tap into the field, we encounter a "zoom" property that I'm assuming is a "helper" iOS provides by default. This [stackoverflow question](https://stackoverflow.com/questions/2989263/disable-auto-zoom-in-input-text-tag-safari-on-iphone) appears to provide a solution in the form of adding `maximum-scale=1` as a meta tag I believe.

**Comment by g-a-v-i-n** [comment]
Noticing a few details

Screen Shot 2021-09-16 at 11 21 15 AM
Toggle switches should be top-aligned

Screen Shot 2021-09-16 at 11 21 09 AM
IMO - these cards should eb gray-50

Screen Shot 2021-09-16 at 11 21 03 AM
Clicking the notifications button when notifications is open should close notifications
